images/stable-baselines/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG on
+FROM python:${on}
+
+RUN apt-get -qy update && \
+    apt-get -qy install libgl1 && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get -qy clean
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+# Copy arena to tmp since bind-mount is read-only and pip doesn't support
+# out-of-tree builds.
+RUN --mount=target=/usr/src/arena,type=bind,source=. \
+    cp -r /usr/src/arena /tmp/arena && \
+    pip install /tmp/arena[stable-baselines] && \
```

Comment:
in addition to this, one needs to install the proper TF version, either CPU or GPU. Do we want to add it here?

Reply:
Good point. Do you know what is additionally need for CPU and GPU?

Reply:
@discordianfish adding `pip install tensorflow-1.14.0 && \` should be enough, right?

Reply:
@discordianfish I will test it with 1.15.5 even if they say they support up to 1.15.0 (https://stable-baselines.readthedocs.io/en/master/guide/install.html#prerequisites). I will let you know.